File: Java/434.java
import java.util.Scanner;

/**
 *
 * 434 - Romance en el Palomar - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int prueba = sc.nextInt();
        for (int i = 1; i <= prueba; i++) {
            int palomo = sc.nextInt();
            int paloma = sc.nextInt();

            if (palomo > paloma) {
                System.out.println("PRINCIPIO");
            } else {
                System.out.println("ROMANCE");
            }

        }
    }
}

File: C/434.c
#include <stdio.h>
#include <stdlib.h>

// 434 - Romance en el Palomar - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop;
    scanf("%d", &loop);
    int i;
    int dovesSleeping = 0;
    int ableBoxes = 0;
    for (i = 0; i<loop; i++) {
        scanf("%d", &dovesSleeping);
        scanf("%d", &ableBoxes);
        if (dovesSleeping>ableBoxes) {
            printf("PRINCIPIO\n");
        } else {
            printf("ROMANCE\n");
        }
    }
    return 0;
}
